LanguageFeatures/Parts-with-imports/scope_A08_t02_part2.dart

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion It’s a compile-time error if any file declares an import prefix
+/// with the same base name as a top-level declaration of the library.
+///
+/// @description Check that it is a compile-time error if a part-file declares
+/// an import prefix with the same base name as a top-level declaration.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=enhanced-parts
+
+part of 'scope_A08_t02_part1.dart';
+
+int l1 = 1;
```

Comment:
We will get a compile-time error here because both part1 and part2 have a declaration with the same name at the top level. So we can't know for sure that the conflict with the import prefix is detected, it might just be the conflict with the other top-level declaration.

Perhaps just remove half of these declarations whose name starts with `l` from part1, and the other half from part2?

Reply:
The idea was to test transitivity. Let's simply add another branch to this tree.